.github/workflows/lighthouse-ci.yml

```diff
@@ -8,8 +8,13 @@ env:
 
 jobs:
     lhci:
-        name: Lighthouse CI
+        name: Lighthouse CI - ${{ matrix.config }}
```

Comment:
Is there a attribute we can use here that just gives us the file name or the name of the config? The whole url is a bit long and makes it difficult to differentiate between on the list of actions